### PR TITLE
Feature/modal news stories

### LINKED ISSUE #60: Take Action Modal
- [ ] Fix Take action modal so that it only shows one action at a time 
- [ ] Add styling to h2s so arrows do not cover

### PR DESCRIPTION
**Pull request by jordan-sullivan**
Hi Team, Please review this PR. 

## Description of Changes:
* Adds functionality to modal in News Stories component
* Adds team contributor names and Github profiles to About section
* Adjusts html/JSX attributes to increase LightHouse score from 80 to 95

## Put an 'X' next to all that apply
New Test: []
New Feature:[x]
Bug Fix: [x]

## Before You Submit Ensure
All Tests Pass: [x]
Full Coverage: [x]
Runs Locally: [x]

Closes #60 

Thanks!

### CODE REVIEW

File: src/components/NewsStories/NewsStories.js [hunk 1]
```diff
@@ -58,58 +58,27 @@ const NewsStories = (event) => {
...
-          </p>
-            <div className="mpo-modal-nav">
-              <label htmlFor="modal-content-2" className="prev-slide">&#8249;&#8249;</label>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
+
+<div id="open-modal" className="modal-window">
```
Comment (camianderson) [inline src/components/NewsStories/NewsStories.js:65]: Good job refactoring the modal! This one looks clean and has less lines of code!

File: src/components/About/About.js [hunk 2]
```diff
@@ -5,12 +5,24 @@ const About = () => {
...
                 This application allows users to see current events on a page, click on one of interest, and view what actionable things they can do to create change around them. Whether it's calling up a local politician to reform laws, writing letters of appreciation to front-line workers in hospitals, or simply meditating to stay grounded , "One Small Thing" is a place of solutions amongst a world of "what can I do to help". <br/><br/>
                 For more information on Makenna, Layla, Maranda, Nevaeh, Jose Manuel, Xavier, Tess, Ellie, Eliahna, Annabell, Jackie, Uziyah, Jayce, Maite, Jailah, Irma, Eva, Amerie, Lexi, Alithia, and their impact in the world, <a target="_blank" href="https://www.texastribune.org/2022/05/25/uvalde-school-shooting-victims/" className="external-link-uvalde" rel="noreferrer">click here.</a> <br/><br/>
                 </p>
+                <h3>BackEnd Team:
```
Comment (camianderson) [inline src/components/About/About.js:14]: Thanks for adding the github links of our team!

File: src/components/NewsStories/NewsStories.js (hunk 1 above)
Comment (jordan-sullivan) [reply src/components/NewsStories/NewsStories.js:65]: Right! I simplified a lot and I think it works better with the cohesiveness of the app. Thanks!

**Review by Neil-B-Linton — APPROVED** [review]
Awesome changes

**Pull request merged**